packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -222,6 +223,11 @@ abstract class ScrollView extends StatelessWidget {
   /// [ScrollBehavior].
   final ScrollBehavior? scrollBehavior;
 
+  /// The speed factor at which scrolling is performed. This is useful when the
+  /// normal speed of scrolling is not enough (or too fast) and needs to be
```

Comment:
```suggestion
  /// normal speed of scrolling is not enough (or too fast) and needs to be
```

This sounds like it is a bug we should fix directly rather than through a parameter like this.